Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -15,6 +15,7 @@
 		75657E4827ABAC8400EE865B /* CompassExampleView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 75657E4727ABAC8400EE865B /* CompassExampleView.swift */; };
 		75C37C9227BEDBD800FC9DCE /* BookmarksExampleView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 75C37C9127BEDBD800FC9DCE /* BookmarksExampleView.swift */; };
 		75D41B2B27C6F21400624D7C /* ScalebarExampleView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 75D41B2A27C6F21400624D7C /* ScalebarExampleView.swift */; };
+		88519D3F2C8A7967007D7015 /* LocationButtonExampleView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88519D3E2C8A7967007D7015 /* LocationButtonExampleView.swift */; };
```

Comment:
I just noticed that when the project file is edited, Xcode automatically moves this line so that it is sorted property:

<img width="555" height="187" alt="image" src="https://github.com/user-attachments/assets/48d0fee5-f127-4f24-a37a-d4aeaf62171d" />

Please fix by making Xcode re-save the project file.